defaults/main.yml

```diff
@@ -7,6 +7,8 @@ ansible_python_interpreter: /usr/bin/python3
 mariadb_use_official_repo: false
 mariadb_use_official_repo_url: https://deb.mariadb.org
 mariadb_use_official_repo_version: 10.6
+# see: https://repolib.readthedocs.io/en/latest/deb822-format.html
```

Comment:
https://manpages.debian.org/stable/apt/sources.list.5.en.html#DEB822-STYLE_FORMAT is official doc.